.github/workflows/release.yml

```
@@ -45,4 +45,7 @@ jobs:
         env:
           TWINE_USERNAME: __token__
           TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
-        run: twine upload --verbose dist/* || echo 'Version exists'
+        shell: bash
+        run:
+          set -e
+          twine upload --skip-existing --verbose dist/*
```

I think this one is no longer necessary. When you remove the `|| echo 'Version exists'` the runner will fail if the twine command fails
```suggestion
          twine upload --skip-existing --verbose dist/*
```